app/models/git/exercism_repo.rb

```diff
@@ -1,4 +1,5 @@
 class Git::ExercismRepo < Git::RepoBase
+  PAGES = %w{about installation tests learning resources}
```

Comment:
I mentioned this in the main conversation, but I would remove `about` here, since it's already included as part of the main track page.